### DISCUSSION TITLE
Missing methods/patterns

### DISCUSSION THREAD

**Discussion by d3vp** [smrg-lm/sc3#14] [General]
So far I have added the following things I found missing in sc3: 
- method: Env.delay
- class Pgauss(ValuePattern)

Is there any alternative to delay?

**Comment by smrg-lm** [comment]
Pgauss is missing along with the patterns in RandomDistPatterns.sc which are part of the main library. It is a TODO, it could go in sc3/seq/patterns/valuepatterns.py under a ```### Random distribution patterns ###``` section at the end.

I don't remember why I didn't transcribe Env.delay, it might be because it's an instance method that returns a new modified instance and was considered sugar. The problem with that kind of methods is that they may pollute the API with particular use cases.
